### PR TITLE
added profile location for NPL

### PR DESCRIPTION
**Pull request by alex-salnikov**
after fiddling with transaction RZ10 (which eded up in breaking my original DEFAULT.PFL file) - I realised that it is much simpler to edit DEFAULT.PFL directly. Maybe this information could helps others?
( my miniSAP is NPL 7.51 / Sybase / openSUSE )

### CODE REVIEW

File: docs/guide-ssl-setup.md [hunk 1]
```diff
@@ -57,3 +57,8 @@ ssl/client_ciphersuites = 150:PFS:HIGH::EC_P256:EC_HIGH
 ssl/ciphersuites = 135:PFS:HIGH::EC_P256:EC_HIGH
 icm/HTTPS/client_sni_enabled = TRUE
 ```
+
+- for NPL minisap - add profile parameters to following file:
```
Comment (mbtools) [inline docs/guide-ssl-setup.md:61]: Maybe add the official title is "SAP NetWeaver AS ABAP Developer Edition on ASE"

Note: "SAP NetWeaver AS ABAP Developer Edition on SAP HANA" has a different sysid/path. I think it's /usr/sap/A4H/sys/profiles. 

https://developers.sap.com/trials-downloads.html?search=abap

**Comment by alex-salnikov** [comment]
- thank you for info on A4H - I have added it to the list.
- regarding providing official titles - I've been looking for a better way of specifying appropriate product title and decided to go with same titles/classification as listed on the [License Keys](https://go.support.sap.com/minisap/#/minisap) page. From that page I understand that for evaluation versions (i.e. miniSAP) there is a unique predefined system-name for particular combination of NetWeaver / database / operating system.

( plz let me know if I can improve this further )

**Review by mbtools — APPROVED** [review]

**Comment by mbtools** [comment]
Great. If you like add the link to the license key page to the docs.

**Comment by alex-salnikov** [comment]
> add the link to the license key page

good idea, thank you .. also sorted the list by system-name alphabetically.

**Comment by larshp** [comment]
thanks @alex-salnikov 

welcome to abapGit as a contributor 🎉

**Pull request merged**